src/binder_nfc_adapter.c

```diff
@@ -584,6 +584,18 @@ gboolean
 binder_nfc_adapter_close(
     BinderNfcAdapter* self)
 {
+    NciCore* nci = self->adapter.nci;
+
+    /* Should never be > NCI_RFST_IDLE but let's check >= just in case */
+    if (nci->current_state >= NCI_RFST_IDLE) {
+        /*
+         * Make sure that state maching isn't going to continue
```

Comment:
A typo? Should this have "state machine"?

Reply:
Of course it's a typo :) Thanks!